### PR TITLE
Update fa-IR.lang

### PR DESCRIPTION
**Pull request by mojtabaatt**
Fix some Persian translate issues

### CODE REVIEW

**Comment by AliMd** [comment]
Thank you, it's better 👍

File: fa-IR.lang [hunk 1]
```diff
@@ -1,10 +1,10 @@
 addOption = اضافه کردن گزینه
 allFieldsRemoved = همه فیلدها حذف شدند.
-allowMultipleFiles = اجازه‌ی انتخاب چندگانه‌ی فایل به کاربران
+allowMultipleFiles = اجازه‌ی انتخاب چند فایل به کاربر
 autocomplete = تکمیل خودکار
 button = دکمه
 cannotBeEmpty = این فیلد نمی تواند خالی باشد
-checkboxGroup = گروه چک باکس
+checkboxGroup = چند انتخابی
```
Comment (AliMd) [inline fa-IR.lang]: `گروه چک باکس` is better

File: fa-IR.lang [hunk 2]
```diff
@@ -28,58 +28,58 @@ fieldDeleteWarning: false,
...
 placeholders.text = یک متن وارد کنید
 placeholders.textarea = متن طولانی خود را وارد کنید
 placeholders.email = ایمیل خود را وارد کنید
-placeholders.placeholder = متن نمایشی
+placeholders.placeholder = متن راهنما
 placeholders.className = کلاس‌ها (با فاصله آنها را جدا کنید)
 placeholders.password = پسورد خود را وارد کنید
 preview = پیش‌نمایش
-radioGroup = رادیو گروهی
+radioGroup = تک انتخابی
```
Comment (AliMd) [inline fa-IR.lang]: Please find better one, both are not usual

Comment (kevinchappell) [reply fa-IR.lang]: @AliMD is this good to merge or needs improvement?

File: fa-IR.lang (hunk 1 above)
Comment (kevinchappell) [reply fa-IR.lang]: I would have to agree, `گروه چک باکس` is more literal (which i like) but if there is not a literal version for radio group, would it make sense to change to `چند انتخابی` for checkboxGroup and `تک انتخابی` for radioGroup?

**Comment by kevinchappell** [comment]
Sorry I'm late to the party. These changes look good to me but you guys would know better.

**Pull request merged**